taichi/util/statistics.cpp

```diff
@@ -28,8 +28,6 @@ void Statistics::print(std::string *output) {
 
 void Statistics::clear() {
   counters.clear();
-  counters["codegen_statements"] = 0;
-  counters["codegen_offloaded_tasks"] = 0;
```

Comment:
I'll be using `dict(...).get('codegen_kernel_statements', 0)` so this is no longer necessary.

Reply:
OK sounds good